snapshotter/config/config.go

```diff
@@ -59,7 +59,14 @@ type debug struct {
 }
 
 type metrics struct {
-	Enable bool `toml:"enable" default:"false"`
+	Enable           bool             `toml:"enable" default:"false"`
+	Host             string           `toml:"host"`
+	PortRange        string           `toml:"port_range"`
+	ServiceDiscovery serviceDiscovery `toml:"service_discovery"`
+}
+
+type serviceDiscovery struct {
+	Port int `toml:"port"`
```

Comment:
How about having ServiceDiscoveryPort on metrics? Would serviceDiscovery have more fields later?

Reply:
sure can do, serviceDiscovery may have more fields later but at this time I don't anticipate any.